pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
```

Comment:
Do we need to add this file and `conftest.py` to the project root? Is there any other way to do it in order to avoid polluting the project root?

Reply:
I've deleted `conftest.py`. However I opt for leaving `pytest.ini` in project root. 
* The `pytest.ini` file allows pytest to indentify project root. 
* It's good clue for developer that we're using `pytest` for running tests.
* It modifies how `pytest` command works, especially in our case it may be not obvious (because it's configured to skip many tests), so it's good to have it in visible place
* Other big projects also keep `pytest.ini` in root dir, for example the [python requests](https://github.com/psf/requests) library

I have never seen a project which would keep `pytest.ini` in different place than project root, I guess it may be also tricky to show pytest proper root_dir in such case.
As I can see we also have `circle.yml` in the project root, so `pytest.ini` is the same kind of file.

Reply:
An alternative could be also `tox.ini` file.